src/authentication/components/StamboekInput.tsx

```diff
@@ -50,37 +50,40 @@ export const StamboekInput: FunctionComponent<StamboekInputProps> = ({
 			status: undefined,
 		},
 		INVALID_FORMAT: {
-			message:
-				'Het stamboek nummer heeft een ongeldig formaat. Geldige formaten: 00000000000 of 00000000000-000000',
+			message: t(
+				'Het stamboek nummer heeft een ongeldig formaat. Geldige formaten: 00000000000 of 00000000000-000000'
+			),
 			status: ToastType.DANGER,
 		},
 		INVALID_NUMBER: {
 			message: (
 				<span>
-					Het stamboek nummer is niet geldig, of nog niet geactiveerd. Indien u nog maar recent uw
-					kaart heeft ontvangen kan u via{' '}
+					<Trans>
+						Het stamboek nummer is niet geldig, of nog niet geactiveerd. Indien u nog maar recent uw
+						kaart heeft ontvangen kan u via
+					</Trans>{' '}
 					<Button
 						onClick={() => redirectToClientPage(APP_PATH.MANUAL_ACCESS_REQUEST, history)}
 						label={t('authentication/components/stamboek-input___een-manuele-aanvraag')}
 						type="inline-link"
 					/>{' '}
-					toch al toegang krijgen.
+					<Trans>toch al toegang krijgen.</Trans>
```

Comment:
Als je dit in 2 aparte Trans tags steekt, verlies je dan niet een beetje de context?

Reply:
idd, helaas kun je in een trans vertaling geen react componenten gebruiken enkel simpele <a href>.

Ik heb de code aangepast om de button na de text te zetten